### ISSUE TITLE
Add the response headers to the thrown WebapiError

### ISSUE THREAD

**Issue by wonkoRT** [thelinmichael/spotify-web-api-node#316]
It would be very useful to add the response headers to the thrown WebapiError. Atm i just need access to "retry-after" but think there might be other uses for the headers as well, if not now maybe in the future.

I need access to "retry-after" when hitting the rate limit. It occurs for me when i execute multiple search queries in parallel using await Promise.all.

Took me 5 minutes to patch http-manager.js and webapi-error.js but it would be great if the official npm could include this feature.

For completeness sake, here's the link to the official docs: [spotify rate limiting](https://developer.spotify.com/documentation/web-api/#rate-limiting)

_Originally posted by @wonkoRT in https://github.com/thelinmichael/spotify-web-api-node/issues/217#issuecomment-633518106_

**Comment by thelinmichael** [comment]
Hey @wonkoRT. Apologies for the late reply. The latest version of the client has a `headers` field in the error response, allowing you to read the `Retry-After` header.
